Sources/MapboxDirections/Incident.swift

```diff
@@ -21,46 +20,174 @@ public struct Incident: Codable, Equatable {
         case geometryIndexStart = "geometry_index_start"
         case geometryIndexEnd = "geometry_index_end"
     }
-
-    var identifier: String
-    var type: String
-    var description: String
-    var creationTime: String
-    var startTime: String
-    var endTime: String
-    var impact: String?
-    var subtype: String?
-    var subtypeDescription: String?
-    var alertCodes: [Int]
-    var lanesBlocked: [Int]
-    var geometryIndexStart: Int
-    var geometryIndexEnd: Int
+    
+    /// Defines known types of incidents.
+    ///
+    /// Each incident may or may not have specific set of data, depending on it's `kind`
+    public enum Kind: String {
+        case Accident = "accident"
+        case Congestion = "congestion"
+        case Construction = "construction"
+        case DisabledVehicle = "disabled_vehicle"
+        case LaneRestriction = "lane_restriction"
+        case MassTransit = "mass_transit"
+        case Miscellaneous = "miscellaneous"
+        case OtherNews = "other_news"
+        case PlannedEvent = "planned_event"
+        case RoadClosure = "road_closure"
+        case RoadHazard = "road_hazard"
+        case Weather = "weather"
+    }
+    
+    /// Defines a lane affected by the `Incident`
+    ///
+    /// Each `Incident` may have arbitrary of affected lanes
+    public enum BlockedLane: String, Codable {
+        case Left = "LEFT"
+        case LeftCenter = "LEFT CENTER"
+        case LeftTurnLane = "LEFT TURN LANE"
+        case Center = "CENTER"
+        case Right = "RIGHT"
+        case RightCenter = "RIGHT CENTER"
+        case RightTurnLane = "RIGHT TURN LANE"
+        case Hov = "HOV"
```

Comment:
“HOV” is an initialism for “high-occupancy vehicle”. I think we should write it out because it’s a somewhat specialized term that especially non-native English speakers may be unfamiliar with.